activation/activation.go

```diff
@@ -527,9 +527,11 @@ func (b *Builder) PublishActivationTx(ctx context.Context) error {
 
 	challenge, err := b.loadChallenge()
 	if err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
+			logger.With().Error("failed to load atx challenge", log.Err(err))
+		}
```

Comment:
I don't understand. This branch will be entered when the file exists and it is corrupted. The change is to not log when the file simply doesn't exist.

Reply:
OK, let me rephrase: This is not an error, the node doesn't behave differently because it couldn't load the ATX challenge (it does the same no matter if there is no file or a corrupted file). Imo the appropriate logging level is Warn or Debug, because any error doesn't change anything in the flow of the program.

Reply:
For sure it must not be a `Debug`. A corrupted nipost_challenge.bin is a serious thing and the user must be made aware of it. Whether Error or Warn is better, IDK. What are the guidelines here?